x-pack/legacy/plugins/maps/public/connected_components/layer_addpanel/import_editor/view.js

```diff
@@ -5,13 +5,14 @@
  */
 
 import React, { Fragment } from 'react';
-import { GeojsonFileSource } from '../../../layers/sources/client_file_source';
+import { getLayerWizard } from '../../../layers/layer_wizard_registry';
 import { EuiSpacer, EuiPanel, EuiButtonEmpty } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
+import { GEOJSON_FILE } from '../../../../common/constants';
 
 export const ImportEditor = ({ clearSource, isIndexingTriggered, ...props }) => {
   const editorProperties = getEditorProperties({ isIndexingTriggered, ...props });
-  const editor = GeojsonFileSource.renderEditor(editorProperties);
+  const layerWizard = getLayerWizard(GEOJSON_FILE);
```

Comment:
We know exactly which one this is, so why not just import it from the module https://github.com/nreese/kibana/blob/61068682610627307b517c7d8caefad136678a8f/x-pack/legacy/plugins/maps/public/layers/sources/client_file_source/geojson_file_source.js#L139 iso of the additional indirection (?)